saleor/graphql/account/i18n.py

```diff
@@ -43,6 +43,10 @@ def validate_address_form(cls, address_data: dict, instance=None):
 
     @classmethod
     def validate_address(cls, address_data: dict, instance=None, info=None):
+        if address_data.get("country") is None:
+            raise ValidationError(
+                {"country": ValidationError("This field is required.", code="required")}
```

Comment:
Isn't there a pattern to use enums as error codes?

Reply:
Yes, but we don't have `AddressErrorCode`. In some universal validation, we using strings instead of enums. E.g. `saleor/graphql/core/utils::validate_image_file`

Reply:
We should have a base enum for all error codes that we could use in cases like this.